modin/pandas/base.py

```diff
@@ -2528,7 +2528,9 @@ def to_csv(
         }
         new_query_compiler = self._query_compiler
 
-        from modin.data_management.factories.dispatcher import FactoryDispatcher
+        from modin.core.execution.dispatching.factories.dispatcher import (
+            FactoryDispatcher,
+        )
```

Comment:
cannot we move this to the top of the file?

Reply:
Even if we can, this should go in a separate issue/PR

Reply:
Agree, please open a new issue for these.